src/dials/util/image_viewer/mask_frame.py

```diff
@@ -691,11 +694,11 @@ def OnLeftDown(self, event):
                 )
                 p1, p0, p_id = self._pyslip.tiles.flex_image.picture_to_readout(yc, xc)
 
-                if p_id < 0:
-                    return
+                # if p_id < 0:
+                #    return
```

Comment:
Little bit of leftover clutter here, but otherwise LGTM

Reply:
Thanks! These lines and the next `if False` block are unnecessary. I will fix it.